pokemongo_bot/cell_workers/move_to_fort_worker.py

```diff
@@ -41,7 +41,13 @@ def work(self):
                 )
 
 
-            if not self._step_walker.step():
+        step_walker = self._step_walker
+            if step_walker == True:
+                self.api.player_update(latitude=lat, longitude=lng)
```

Comment:
please just add this `self.api.player_update` calls  after each step is taken by the `StepWalker` and revert all the other changes in this file.